modules/UI.js

```diff
@@ -0,0 +1,34 @@
+import Store from './store.js';
+// UI for UI tasks
+export default class UI {
+  static displayBooks = () => {
+    const books = Store.getBooks();
+
+    books.forEach((book) => UI.addBook(book));
+  }
+
+  static addBook = (book) => {
+    const list = document.querySelector('.table-body');
+
+    const row = document.createElement('tr');
+
+    row.innerHTML = `
+                  <td>${book.title}</td>
+                  <td>by${book.author}</td>
+                  <td><a href="#" class="remove-btn">Remove</a></td>
+              `;
+
+    list.appendChild(row);
+  }
+
+  static deleteBook = (el) => {
+    if (el.classList.contains('remove-btn')) {
+      el.parentElement.parentElement.remove();
```

Comment:
As suggested by the previous code reviewer  You need to display the current date below the navigation bar, and for this, you need to use the Luxon library. Kindly go through project requirements and implement this feature.